include/zipped_rdd.h

```diff
@@ -0,0 +1,64 @@
+#ifndef CPARK_ZIPPED_RDD_H
+#define CPARK_ZIPPED_RDD_H
+
+#include <iterator>
+#include <ranges>
+#include <vector>
+
+#include "base_rdd.h"
+#include "utils.h"
+
+namespace cpark {
+
```

Comment:
The same requirements go here.
```suggestion

/** @ingroup g_trans
 *  @defgroup t_Zip Zip
 *  This forms the Zip Transformation of our cpark library
 *  @image html zip.drawio.png "ZippedRdd diagram" width=50%
 *  @image latex zip.drawio.png "ZippedRdd diagram" width=10cm
 *  @see ZippedRdd
 *  @{
 */
```

Reply:
Got it. Will refer to the existing main repo to add the comments